astrodb_utils/publications.py

```diff
@@ -267,117 +250,52 @@ def ingest_publication(
         logger.error("Publication, DOI, or Bibcode is required input")
         return
 
-    if not ignore_ads:
-        ads_token = check_ads_token()
-
-        if not ads_token:
-            logger.warning(
-                "An ADS_TOKEN environment variable is not set.\n"
-                "setting ignore_ads=True.")
-            ignore_ads = True
-
-            if (not reference and (not doi or not bibcode)):
-                logger.error(
-                    "An ADS_TOKEN environment variable must be set"
-                    "in order to auto-populate the fields.\n"
-                    "Without an ADS_TOKEN, name and bibcode or DOI must be set explicity."
-                )
-                return
-   
+    if reference and not bibcode and not doi:
```

Comment:
In general, I recommend if statements be explicit unless they are booleans already. Neither reference, bibcode, or doi are booleans so you would want to use things like `if reference is not None` instead of just `if reference`